meson.build

```diff
@@ -89,12 +89,12 @@ test_link_args = [
   '-Wl,-z,relro',
   '-Wl,-z,now',
 ]
-foreach arg: test_link_args
-  if cc.has_link_argument(arg)
-    global_link_args += arg
+foreach link_arg: test_link_args
+  if cc.links('int main () { return 0; }', name: link_arg, args: link_arg)
```

Comment:
is this a workaround? Seems a little hacky...

Reply:
You are right, this is a hack, I did not notice has_link_argument was added in 0.45.